stix_shifter_modules/onelogin/test/stix_translation/test_onelogin_stix_to_query.py

```diff
@@ -227,6 +227,12 @@ def test_risk_cookie_id_query(self):
         queries = 'risk_cookie_id=5&limit=1000'
         _test_query_assertions(query, queries)
 
+    def test_in_operator(self):
+        stix_pattern = "[x-onelogin-finding:operation_name IN ('Default', 'notDefault') ]"
+        query = translation.translate('onelogin', 'query', '{}', stix_pattern)
+        queries = 'operation_name=Default or operation_name=notDefault &limit=1000'
```

Comment:
does this translated query works? I see few whitespaces in the query string which might fail the api query. also you need to confirm whether this ` or `  operator supports by the api.